bundle/metadata/metadata.go

```diff
@@ -0,0 +1,45 @@
+package metadata
+
+import (
+	"github.com/databricks/cli/bundle/config"
+)
+
+var Version = 1
+
+type Bundle struct {
+	Git config.Git `json:"git,omitempty"`
+}
+
+type Workspace struct {
+	FilesPath string `json:"file_path,omitempty"`
+}
+
+type Job struct {
+	ID string `json:"id,omitempty"`
+
+	// Relative path from the bundle root to the configuration file that holds
+	// the definition of this resource.
+	RelativePath string `json:"relative_path,omitempty" bundle:"readonly"`
```

Comment:
The bundle tag is not used for this set of structs (not relevant for the schema).